spec2vec/Spec2Vec.py

```diff
@@ -1,18 +1,58 @@
+from typing import Union
 import scipy
 from .calc_vector import calc_vector
 
 
 class Spec2Vec:
+    """Calculate spec2vec similarity scores between a reference and a query.
 
-    def __init__(self, model, intensity_weighting_power=0):
+    Using a trained model, spectrum documents will be converted into spectrum
+    vectors. The spec2vec similarity is then the cosine similarity score between
+    two spectrum vectors.
+    """
+    def __init__(self, model, intensity_weighting_power=0,
+                 allowed_missing_fraction: Union[float, int] = 0):
+        """
+
+        Parameters
+        ----------
+        model : gensim word2vec model
+            Expecgted input is a gensim word2vec model that has been trained on
+            the desired set of spectrum documents.
+        intensity_weighting_power : float, optional
+            Spectrum vectors are a weighted sum of the word vectors. The given
+            word intensities will be raised to the given power.
+            The default is 0, which means that no weighing will be done.
+        allowed_missing_fraction:
```

Comment:
If you name a var fraction I expect to use 0..1 a fraction as value. Could you either use a real fraction everywhere or change var name to `allowed_missing_percentage`?

Reply:
Changed it to ``allowed_missing_percentage``.